### PR TITLE
Adds Nmajask to the game

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

She's been maintainer for uhh idk 

# Wiki Documentation

we have a list of maintainers on wiki

:cl:  
rscadd: Nmajask is in the game now
/:cl:

### CODE REVIEW

**Comment by nmajask** [comment]
Bad PR

**Comment by Chubbygummibear** [comment]
next i want a nmajask plush in the modular console vendor that you can only get by inserting a coin for the premium items

File: html/changelog.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		<td valign='top'>
 			<font size='2'><b>Current Head Developers:</b> JamieD12, Theos<br></font>
 			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Maintainers:</b> AshCorr, Identification, Monster860, Partheo, Asd, TheGamerdk, adamsogm, baiomu<br></font>
+			<font size='2'><b>Maintainers:</b> AshCorr, Identification, Monster860, Partheo, Asd, TheGamerdk, adamsogm, baiomu, Nmajask<br></font>
```
Comment (adamsong) [inline html/changelog.html:40]: This is overwritten by header.html

**Review by adamsong — DISMISSED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by Moja10** [comment]
> Bad PR

Can we unmerge? They don't want it. Do not lecture me about the word unmerge

File: html/changelog.html (hunk 1 above)
Comment (ToasterBiome) [reply html/changelog.html:40]: > This is overwritten by header.html

should it be removed?

**Comment by ToasterBiome** [comment]
> > Bad PR
> 
> Can we unmerge? They don't want it. Do not lecture me about the word unmerge

i do believe this is what someone calls a joke

**Pull request merged**